pkg/reconciler/pipelinerun/resources/apply_test.go

```diff
@@ -3920,15 +3954,6 @@ func TestApplyTaskResultsToPipelineResults_Error(t *testing.T) {
 		},
 		expectedResults: nil,
 		expectedError:   fmt.Errorf("invalid pipelineresults [foo], the referred results don't exist"),
-	}, {
-		description: "unsuccessful-taskrun-no-returned-result",
-		results: []v1beta1.PipelineResult{{
-			Name:  "foo",
-			Value: *v1beta1.NewStructuredValues("$(tasks.pt1.results.foo)"),
-		}},
-		taskResults:     map[string][]v1beta1.TaskRunResult{},
-		expectedResults: nil,
-		expectedError:   fmt.Errorf("invalid pipelineresults [foo], the referred results don't exist"),
```

Comment:
Shouldn't we "update" that Test, the expected Error should, however change (not invalid result, but something else) ? But maybe this is not at the same level.

Reply:
I don't remember why I removed this test, could be a mistake

Reply:
added this back